tests/spirv_hopper/CMakeLists.txt

```
@@ -13,7 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ~~~
-add_subdirectory(external)
+add_library(spirv-reflect STATIC)
+target_sources(spirv-reflect PRIVATE
+    ${SPIRV_REFLECT_INSTALL_DIR}/spirv_reflect.h
+    ${SPIRV_REFLECT_INSTALL_DIR}/spirv_reflect.cpp
+)
```

Since the include directory belongs to spirv-reflect it should be a PUBLIC include of spirv-reflect.

```suggestion
)

target_include_directories(spirv-reflect SYSTEM PUBLIC ${SPIRV_REFLECT_INSTALL_DIR})
```